src/behavior/src/A32NX_Interior_Handling.xml

```diff
@@ -623,4 +623,22 @@
             </UseTemplate>
     </Template>
 
+    <Template Name="FBW_HANDLING_Switch_AntiSkid_Template">
+        <DefaultTemplateParameters>
+            <WWISE_EVENT_1>lswitch</WWISE_EVENT_1>
+            <WWISE_EVENT_2>lswitch</WWISE_EVENT_2>
+        </DefaultTemplateParameters>
+        <Component ID="#NODE_ID#" Node="#NODE_ID#">
+            <UseTemplate Name="ASOBO_GT_Switch_Code">
+                <LEFT_SINGLE_CODE>
+                    (&gt;K:ANTISKID_BRAKES_TOGGLE)
+                    (A:ANTISKID BRAKES ACTIVE, Bool) ! if{
+                        0 (&gt;L:XMLVAR_Autobrakes_Level)
+                    }
+                </LEFT_SINGLE_CODE>
```

Comment:
Has this code been discussed with @crocket63?

Reply:
Nope :)  XMLVAR_Autobrakes_Level is not used any more and removed from code. What is the functional need here?

Reply:
If the aim of this is just to disable autobrake once button is switched off, it can be simply removed as new autobrake will process this already in wasm part by looking at ANTISKID BRAKES ACTIVE simvar

Reply:
I just imported the template we were using for the switch, the same code that was used before but instead of calling on asobos template I moved it to interior_handeling, changed the name of the template and sound event, if it's not used I can just remove that, but it was calling for that template out of our code before as well.

Reply:
Mmmm maybe I'm doing something wrong but I can't find the XMLVAR anywhere doing a search in last dev.

Reply:
It was not in the mod it was looking for a template in official adobo packages, look for : ASOBO_HANDLING_Switch_AntiSkid_Template

Reply:
Pushed a commit that reduces that to `<LEFT_SINGLE_CODE>(&gt;K:ANTISKID_BRAKES_TOGGLE)</LEFT_SINGLE_CODE>` since that part is still needed for the switch to function.

Reply:
Just need to make sure this still makes ANTISKID BRAKES ACTIVE change its state, then it's good :)

Reply:
Just checked, it does toggle ANTISKID BRAKES ACTIVE it also triggers the ecam and everything so we should be fine with only that one line of LEFT_SINGLE_CODE